tests/fixtures/issues/issue-865.php

```diff
@@ -2545,59 +2545,6 @@
         'lite' => false,
         'standard' => true,
     ],
-    'issue-865-AW' => [
-        'ua' => 'Mozilla/5.0 (Windows NT 6.3; rv:36.0) Gecko/20100101 Firefox',
```

Comment:
Ultimately I ended up removing this one from the test suite. I played with changing the Generic pattern to not have the slash after "Firefox" (and just be `Firefox*`) but this caused a lot of other issues since it shortened the generic pattern too much. I could have added a pattern just for this useragent so that it would still detect as Firefox (no version), but it didn't seem worth it.